spec/tictactoe_spec.rb

```diff
@@ -1,9 +1,16 @@
 require "tictactoe"
 
 RSpec.describe Tic_Tac_Toe do
-  describe 'run_app' do
-    it 'returns the sum of its arguments' do
-      expect(Tic_Tac_Toe.new.run_app()).to eq("Welcome to Tic-Tac-Toe")
+  
+  describe "#show_empty_board" do
+    it "shows empty TTT board in terminal" do
+      new_game = Tic_Tac_Toe.new
+      row = "   " +  "|" +  "   " + "|" + "   "
+      separator = "-----------"
+      board = row + "\n" + separator + "\n" + row + "\n" + separator + "\n" + row
```

Comment:
Now that the puts and stdout has been removed. I notice that this test setup is the same as the code being tested. Let's make the test setup more verbose, so that someone looking at the test can easily visualize what an empty board will look like.
```suggestionboard  = "   |   |   \n"
      board += "-----------\n"
      board += "   |   |   \n"
      board += "-----------\n"
      board += "   |   |   "
```

Reply:
👍🏽